test/unit/mocks/MediaPlayerModelMock.js

```diff
@@ -169,10 +169,6 @@ class MediaPlayerModelMock {
         }
     }
 
-    getStableBufferTime() {
```

Comment:
some attributes can be removed from MediaPlayerModelMock like retryAttempts, stableBufferTime, etc....

Reply:
It was already cleaned up. I left retryAttempts methods because they add some extra logic that that depends on low latency mode (same for stableBufferTime).